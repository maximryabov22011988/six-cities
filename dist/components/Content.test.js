'use strict';
Object.defineProperty(exports, '__esModule', { value: true });
const React = require('react');
const renderer = require('react-test-renderer');
const Content_1 = require('./Content');
it('Content correctly render', () => {
  const tree = renderer
    .create(React.createElement(Content_1.default, { isEmpty: false, parentClassName: 'favorites' }, null))
    .toJSON();
  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Content.test.js.map
