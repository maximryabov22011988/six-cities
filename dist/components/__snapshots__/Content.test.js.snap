// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content correctly render 1`] = `
<main
  className="page__main page__main--favorites"
>
  <div
    className="container page__favorites-container"
  >
    <section
      className="favorites"
    />
  </div>
</main>
`;
